content/desktop-environment.md

```diff
@@ -43,6 +43,8 @@ If multiple desktop environments are installed, GDM will display a gear icon, wh
 
 ![Login2](/images/desktop-environment/Login2.png)
 
+If you do not find the gear icon to switch the desktop environment, click the user icon first to see the gear button.
```

Comment:
"Do not find" isn't a very natural English expression. It should either be "do not see" ("don't see") or "can't find."

Reply:
I agree. Thanks.